Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -494,6 +495,7 @@ open class MapView: UIView {
         guard notification.object as? UIScene == window?.parentScene else { return }
 
         displayLink?.isPaused = true
+        mapboxMap.reduceMemoryUse()
     }
```

Comment:
I think we only subscribe to these notifications when we have a non-nil display link. Do we need to change that in light of this addition?